superset/assets/package.json

```diff
@@ -56,7 +57,7 @@
     "jquery": "^3.2.1",
     "jsdom": "9.12.0",
     "lodash.throttle": "^4.1.1",
-    "mapbox-gl": "^0.26.0",
+    "mapbox-gl": "^0.27.0",
```

Comment:
I'm guessing this was an attempt at fixing the `npm install` bug. It's been fixed on master by using `yarn` instead of `npm install` so this is probably not needed anymore. Though if you've tested that the mapbox viz renders alright I'm not against the bump here even though it would belong in another PR ideally.